libsrc/include/SimConnect.hpp

```diff
@@ -4,17 +4,17 @@
 //
 //-----------------------------------------------------------------------------
 
-#include "Windows.h"
+#include <windows.h>
```

Comment:
Please remove all formatting changes and make this lowercase change the only diff in the file.

Also for the record, [this stack overflow answer](https://stackoverflow.com/a/15466951) also provides reasoning and supports the change.